opm/upscaling/UpscalerBase_impl.hpp

```diff
@@ -296,24 +296,24 @@ namespace Opm
 	double side1_area = 0.0;
 	double side2_area = 0.0;
 
-	int num_faces = 0;
-	int num_bdyfaces = 0;
-	int num_side1 = 0;
-	int num_side2 = 0;
+//	int num_faces = 0;
+//	int num_bdyfaces = 0;
+//	int num_side1 = 0;
+//	int num_side2 = 0;
```

Comment:
I think we should just remove these, and the associate debugging statement, altogether.  If someone needs the values for debugging/analysis purposes they can make local changes to their copies.

Reply:
sure. just didn't want to invalidated the commented cout line, but i agree it is better to remove altogether.